IntercomDotnet/Resources/Users.cs

```diff
@@ -47,7 +47,7 @@ public dynamic DeleteWithUserId(string userId)
         [Obsolete("userId is a string in the Intercom API so please use the string-based overload going forward.")]
         public dynamic Get(string email = null, int? userId = null)
         {
-            return Get(email, userId?.ToString());
+            return Get(email, Convert.ToString(userId));
```

Comment:
This will return an empty string rather than `null` for a null input, but I see from [here](https://github.com/tarr11/IntercomDotnet/pull/24/files#diff-ebd0c2e41406e094824ef0a7d4a2efbbR60) that is OK, seeing as we check for `IsNullOrEmpty` - so I think this is OK to merge